certora/specs/StakingRouterInvariants.spec

```diff
@@ -96,7 +98,7 @@ function differentOrEqualToZero_Address(address a, address b) returns bool {
 
 function safeAssumptions(uint256 moduleId) {
     requireInvariant modulesCountIsLastIndex();
-    if(moduleId > 0) {
+    if(moduleId > 0) { //this is strange if you invariants are prove - you can safely assume them 
```

Comment:
One of the invariants here leads to vacuity if the entry is zero.

Reply:
Exactly, need to fix this